testing/dart/.gitignore

```diff
@@ -1,2 +1,3 @@
+.dart_tool
```

Comment:
Add this to the root gitignore maybe?

Reply:
Done in follow-up https://github.com/flutter/engine/pull/5019